src/flag_gems/runtime/error/error.py

```diff
@@ -0,0 +1,31 @@
+import logging
+
+
+def PASS(e):
+    pass
+
+
+class ErrorHandler:
```

Comment:
You don't need a class to collect these functions. It does not even have meaningful data members.

If you want rich-typed error, subclass Exception and define custom error.